docs/content/preview/explore/transactions/explicit-locking.md

```diff
@@ -99,3 +101,77 @@ COMMIT
 ```
 
 This should succeed.
+
+## Advisory locks
+
+{{<tags/feature/tp>}}Advisory locks are available in v2.25.1 and later.
+
+YSQL also supports advisory locks, where the application manages concurrent access to resources through a cooperative locking mechanism. Advisory locks can be less resource-intensive than table or row locks for certain use cases because they don't involve scanning tables or indexes for lock conflicts. They are session-specific and managed by the client application.
+
+In PostgreSQL, if an advisory lock is taken on one session, all sessions should be able to see the advisory locks acquired by any other session. Similarly, in YugabyteDB, if an advisory lock is acquired on one session, all the sessions should be able to see the advisory locks regardless of the node the session is connected to. This is achieved via the pg_advisory_locks system table, which is dedicated to hosting advisory locks. All advisory lock requests are stored in this system table.
+
```

Comment:
Are there changes to pg_locks view? eg, locktype can now also be advisory?

Reply:
pg_locks changes are planned, but its not yet in the master builds.